x-pack/plugins/maps/public/classes/styles/vector/components/vector_style_editor.tsx

```diff
@@ -392,8 +399,10 @@ export class VectorStyleEditor extends Component<Props, State> {
           <VectorStyleIconEditor
             disabled={!hasMarkerOrIcon}
             disabledBy={VECTOR_STYLES.ICON_SIZE}
+            customIcons={this.props.customIcons}
             onStaticStyleChange={this._onStaticStyleChange}
             onDynamicStyleChange={this._onDynamicStyleChange}
+            onCustomIconsChange={this._onCustomIconsChange}
```

Comment:
Just make `onCustomIconsChange={this.props.onCustomIconsChange}` and remove _onCustomIconsChange